lavague-integrations/drivers/lavague-drivers-selenium/lavague/drivers/selenium/base.py

```diff
@@ -44,7 +47,7 @@ def default_init_code(self) -> Any:
         from selenium.webdriver.common.keys import Keys
         from selenium.webdriver.common.action_chains import ActionChains
 
-        chrome_options = Options()
+        chrome_options = self.options or Options()
         if self.headless:
```

Comment:
I think it would be better that the "easy" options, such as headless or even the loading strategy options are ignored in case a Chrome Option object is used, as you might want to specify them yourself. What do you think?

Reply:
Yeah, that makes sense, as the user provided the chrome_options, we do not need to look at the other options in the parameter as the user might specify by themselves. So to be sure, the code I need to change is the following:

```python
if self.options:
    chrome_options = self.options
else:
    chrome_options = Options()
    if self.headless:
        chrome_options.add_argument("--headless")
    if self.user_data_dir:
        chrome_options.add_argument(f"--user-data-dir={self.user_data_dir}")
    chrome_options.add_argument("--no-sandbox")
    chrome_options.page_load_strategy = (
        "normal" if self.no_load_strategy is False else "none"
)
```

Please let me know if this is the case, then I'll update the code and make PR again!

Reply:
Yes, this is indeed the case!

Reply:
The requested changes has been made.